compositor_common/src/validators.rs

```diff
@@ -206,6 +212,21 @@ impl SceneSpec {
 
         Ok(())
     }
+
+    fn validate_builtin_transformations(&self) -> Result<(), SpecValidationError> {
+        for spec in &self.nodes {
+            if let NodeSpec {
+                node_id,
+                input_pads,
+                params: NodeParams::Builtin { transformation, .. },
+            } = spec
+            {
+                transformation.validate(node_id, input_pads)?;
+            }
```

Comment:
```suggestion
            let NodeSpec {
                node_id,
                input_pads,
                params,
            } = spec;
            if let NodeParams::Builtin { transformation, .. } = params {
                transformation.validate(node_id, input_pads)?;
            }
```
formating like that looks to me like the block at the end is not a part of an if statement.

Reply:
Ok. It seemed to me straightforward, but I guessed it was because that's how it's usually done in Elixir.
Pattern matching like this seems cleanest to me:
```rust
        for NodeSpec {
            node_id,
            input_pads,
            params,
        } in &self.nodes
        {
            if let NodeParams::Builtin { transformation, .. } = params {
                transformation.validate(node_id, input_pads)?;
            };
        }
```
but I can go with the one you suggested as well (IMO it doesn't matter).

Reply:
sure, that is fine too, with for it's easier to notice, for me that opening `{` is tripping me up